src/seedu/addressbook/data/AddressBook.java

```diff
@@ -61,6 +61,12 @@ public void clear() {
         allPersons.clear();
     }
 
+
+    /**
+     * Sort all persons by name.
+     */
+    public void sort() { allPersons.sort();}
+
```

Comment:
Unnecessary newline at line 64. One newline at line 63 will do

Reply:
The braces and statements in the method should be in separate lines. Refer to coding standard.